Completion/X/Command/_kdeconnect-cli

```diff
@@ -0,0 +1,35 @@
+#compdef kdeconnect-cli
+
+_kdeconnect-cli() {
+  _kdeconnect-cli-devices-id() {
+    compadd $(kdeconnect-cli --list-available --id-only)
+  }
+
+  _arguments \
+    '(-l --list-devices)'{-l,--list-devices}'[List all devices]' \
+    '(-a --list-available)'{-a,--list-available}'[List available (paired and reachable) devices]' \
+    '--id-only[Make --list-devices or --list-available print only the devices id, to ease scripting]' \
+    '--refresh[Search for devices in the network and re-establish connections]' \
+    '--pair[Request pairing to a said device]' \
+    '--ring[Find the said device by ringing it.]' \
+    '--unpair[Stop pairing to a said device]' \
+    '--ping[Sends a ping to said device]' \
+    '--ping-msg:Same as ping but you can set the message to display:' \
+    '--share:Share a file to a said device:_files' \
+    '--list-notifications[Display the notifications on a said device]' \
+    '--lock[Lock the specified device]' \
+    '--send-sms:Sends an SMS. Requires destination:' \
```

Comment:
Like `--ping-msg`, you should use a description of the option in square brackets and a description of the argument after the colon. e.g:
`'--send-sms[send an SMS]:destination'`

or is the destination supposed to be specified with --destination?

Starting descriptions with the command form of the verb is preferred over simple present tense, "send" instead of "sends" in this case. It is also good to be consistent - you have "display" and not "displays" earlier.